tests/client.test.ts

```diff
@@ -46,45 +88,45 @@ test('send and confirm transaction', async () => {
 });
 
 
-test('get epoch info', async () => {
-    {
-        const {epoch, absoluteSlot, slotIndex, slotsInEpoch} = await connection.getEpochInfo();
-        expect(Math.floor(absoluteSlot/slotsInEpoch)).toBe(epoch);        
-    }
-
-    let process_absoluteSlot;
-    {
-        const {epoch, absoluteSlot, slotIndex, slotsInEpoch} = await connection.getEpochInfo({ commitment: 'processed' });
-        expect(Math.floor(absoluteSlot/slotsInEpoch)).toBe(epoch);
-        process_absoluteSlot = absoluteSlot;  
-    }
-
-    let confirmed_absoluteSlot;
-    {
-        const {epoch, absoluteSlot, slotIndex, slotsInEpoch} = await connection.getEpochInfo({ commitment: 'confirmed' });
-        expect(Math.floor(absoluteSlot/slotsInEpoch)).toBe(epoch);      
-        confirmed_absoluteSlot = absoluteSlot;  
-    }
-    expect(confirmed_absoluteSlot >= process_absoluteSlot);
-
-    let finalized_absoluteSlot;
-    {
-        const {epoch, absoluteSlot, slotIndex, slotsInEpoch} = await connection.getEpochInfo({ commitment: 'finalized' });
-        expect(Math.floor(absoluteSlot/slotsInEpoch)).toBe(epoch);        
-        finalized_absoluteSlot = absoluteSlot;  
-    }
-    expect(process_absoluteSlot > finalized_absoluteSlot);
-    expect(confirmed_absoluteSlot > finalized_absoluteSlot);
-
-});
-
-
-test('get leader schedule', async () => {
-    {
-        const leaderSchedule = await connection.getLeaderSchedule();
-        expect(Object.keys(leaderSchedule).length > 0);        
-    }
-});
+// test('get epoch info', async () => {
```

Comment:
commented code - decide what to do with it

Reply:
Not needed really